packages/platform-browser/src/dom/dom_renderer.ts

```diff
@@ -234,9 +234,8 @@ class DefaultDomRenderer2 implements Renderer2 {
   }
 
   setStyle(el: any, style: string, value: any, flags: RendererStyleFlags2): void {
-    if (flags & RendererStyleFlags2.DashCase) {
-      el.style.setProperty(
-          style, value, !!(flags & RendererStyleFlags2.Important) ? 'important' : '');
+    if (flags & (RendererStyleFlags2.DashCase | RendererStyleFlags2.Important)) {
+      el.style.setProperty(style, value, flags & RendererStyleFlags2.Important ? 'important' : '');
```

Comment:
I think that the `if` condition means that `flags & RendererStyleFlags2.Important` is truthy here already. So I think this line could just be:

```suggestion
      el.style.setProperty(style, value, 'important');
```

Reply:
The `if` above covers both `DashCase` and `Important`. If we have `DashCase`, but not `Important`, this should be an empty string.

Reply:
Oops! Yes, sorry.